schema/Mutation.graphql

```diff
@@ -84,4 +84,9 @@ type Mutation {
     Revoke user session
     """
     revokeUserSession(id: ObjectID!, displayNotice: Boolean): Boolean!
+
+    """
+    Create Life
+    """
+    createLife(life: LifeCreateInput): Life!
```

Comment:
the input should be mandatory, don't you agree ?

Reply:
Fixed. Will include in next commit (part 5)